steambox_nvidia/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM steambox
+MAINTAINER Jereme Corrado "jereme@zoion.net"
+
+USER root
+
+RUN apt-get install -y debian-archive-keyring
+RUN echo "deb http://ftp.debian.org/debian/ jessie-backports main contrib non-free" > /etc/apt/sources.list.d/jessie-backports.list
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive \
+    apt-get install -y -t jessie-backports nvidia-driver
```

Comment:
@jcorrado Sounds good.  Some observations below that may help in your experimentation:

If it is true that the docker image's driver must be identical to the host's driver, then it may be challenging to use SteamOS' driver in many cases.  While the docker image can have different shared objects, binaries, and configuration files, it will always use the *same* kernel as the host.  This means that a user would have to (re)compile the SteamOS driver for their host distro's kernel, which may be a different version and have a different configuration than what Valve expected.

Now, for you an me, this will (probably) not be a problem -- SteamOS Brewmaster is based on Debian Jessie.  My host is also Debian Jessie, so I'm using the kernel version that SteamOS expects.  But just to be sure I didn't have problems, I opted to use the proprietary ATI kernel because I knew I could (DKMS) build that driver for both image and host regardless of the kernel version for the most part.

So, in a nutshell, if your kernel version and/or config deviates significantly from SteamOS, then it might be hard to make host and image drivers match.  I still don't know if I believe this will cause a problem, since the docker container is unable to load a driver anyway -- it will just use the one that the host installed.

Then there's that pesky matter of build instructions.  The readme tells people how to build the images, but it doesn't tell them what to do to their host machine.  

Overall, it may be challenging to maintain a portable set of Dockerfiles that also include 3D drivers.  I guess we'll know more as people use this code and complain or ask for help.